docs/guides/continuous-integration/github-actions.mdx

```diff
@@ -302,7 +302,7 @@ on: push
 jobs:
   install:
     runs-on: ubuntu-22.04
-    container: cypress/browsers:node12.18.3-chrome87-ff82
+    container: cypress/browsers:node18.12.0-chrome107
```

Comment:
We should align these updates across our examples: https://github.com/cypress-io/cypress-documentation/pull/5076

We recently re-added the `latest` tag to our docker-images. For the longevity of these examples, thoughts on using the `latest` tag? cc// @elylucas 

```suggestion
    container: cypress/browsers:latest
```